sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCoalesceBatches.scala

```diff
@@ -346,6 +362,8 @@ abstract class AbstractGpuCoalesceIterator(
           if (nextRows > 0) {
             numInputRows += nextRows
             val nextBytes = getBatchDataSize(cb)
+            largestInputBatchSize = largestInputBatchSize.max(nextRows)
```

Comment:
Here it is switched too. So it looks like you are doing the right thing, but the names really confuse me.

Reply:
Yeah that is super confusing.  I could see accidentally switching one, but not sure how I managed to switch both. 🤦